src/js/actions/DialogActions.js

```diff
@@ -0,0 +1,66 @@
+var AppDispatcher = require("../AppDispatcher");
```

Comment:
As we are now exporting an instance of `dispatcher`, we should keep the lowercasing consistent and refactor `AppDispatcher` to `appDispatcher` (including the filename). Will create a PR shortly!